src/referencePackages/src/humanizer.core/2.2.0/lib/netstandard1.0/Humanizer.cs

```diff
@@ -18,6 +18,7 @@
 [assembly: System.Reflection.AssemblyProduct("Humanizer (netstandard1.0)")]
 [assembly: System.Reflection.AssemblyTitle("Humanizer")]
 [assembly: System.Reflection.AssemblyVersionAttribute("2.2.0.0")]
+[assembly: System.Reflection.AssemblyInformationalVersion("2.2.0.0 originated from source-build-reference-packages")]
```

Comment:
This conflicts with the `AssemblyInformationalVersion` attribute on line 17. This is seemingly the case for all packages. Rather than adding a new attribute, I think the existing attribute will need to be modified.

Reply:
Should the text "2.2.0+Branch.master.Sha.b30550eed103a6970d8465fe7c5c16300b70be81" be kept with "originated from source-build-reference-packages" appended to the end of the string, or should I replace the first attribute on line 17 with the new SBRP attribute?

Reply:
Addressed in this commit: https://github.com/dotnet/source-build-reference-packages/pull/794/commits/95dae831be4d21f4443766df377f7ee554ff01b8